#!/bin/sh /usr/share/dpatch/dpatch-run
## 315-magic-add-mdi.dpatch by Daniele Sempione <scrows@oziosi.org>
##
## DP: New magic for Microsoft Document Imaging Format (.mdi) (Closes: #430548).

@DPATCH@

diff -Naurp file-4.23.orig/magic/Magdir/msdos file-4.23/magic/Magdir/msdos
--- file-4.23.orig/magic/Magdir/msdos	2008-01-12 17:11:42.000000000 +0000
+++ file-4.23/magic/Magdir/msdos	2008-01-12 17:11:33.000000000 +0000
@@ -670,3 +670,8 @@
 >16	lelong	>0			%hd x
 >12	lelong	>0			%hd,
 >84	string	x			%.4s
+
+# Type: Microsoft Document Imaging Format (.mdi)
+# URL:  http://en.wikipedia.org/wiki/Microsoft_Document_Imaging_Format
+# From: Daniele Sempione <scrows@oziosi.org>
+0	short	0x5045			Microsoft Document Imaging Format
